metabase/metabase
# Summarize sidebar improvements

[Product doc](https://www.notion.so/metabase/Make-obvious-multiple-groupings-in-GUI-QB-4b788615572c4f928ec462c294da229d)

Please read the product doc for more context.

### Changes to verify the summarization sidebar
- Aggregations under "Summarize by" shown as inline pills
- Add aggregation button changed its appearance
- Dimensions appearance
    - Add dimension button is visually separated
    - Hover state colors have changed
- When a dimension is added, sidebar closed and reopened, the selected dimension becomes pinned to the top of the list
    - If a pinned dimension is from another table, the dimension list item contains the table name
- Everything should work as it worked before

### Screenshot
<img width="371" alt="Screen Shot 2021-10-22 at 02 42 23" src="https://user-images.githubusercontent.com/14301985/138373977-c499bf25-a5b3-474b-abac-55b5754f3b38.png">
